src/modules/LaptopDetail/components/CommentTab/LaptopComment.tsx

```diff
@@ -0,0 +1,179 @@
+import { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import {
+  Rate,
+  Typography,
+  Input,
+  Divider,
+  Pagination,
+  notification
+} from "antd";
+import * as Yup from "yup";
+import { Controller, useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+
+import { CommonButton } from "components";
+import { useAppDispatch, useAppSelector } from "hooks/useRedux";
+import { RootState } from "redux/store";
+import { IComment, ICommentInput } from "types/comment.model";
+import {
+  addNewCommentActionRequest,
+  setNullCommentItem,
+  updateCommentByIdActionRequest
+} from "redux/features/laptop.slice";
+import { IProfile } from "types/profile.model";
+import {
+  CommentList,
+  StarEvaluation,
+  SubmitCommentButton
+} from "./CommentTab.styled";
+import CommentCard from "./CommentCard";
+
+const { TextArea } = Input;
+const { Title } = Typography;
+
+const LaptopComment = () => {
+  const { t } = useTranslation();
+  const [pageNum, setPageNum] = useState<number>(1);
+  const dispatch = useAppDispatch();
+  const { commentList, commentItem, laptopDetail, loading } = useAppSelector(
```

Comment:
(Not required) Should change commentItem to commentID and store ID to Global store

Reply:
I dont't understand. Can you explain more carefully for me?

Reply:
In global store just store ID not an entire object